sql/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -78,7 +78,7 @@ public void testCopyFromFile() throws Exception {
         String uriPath = Joiner.on("/").join(copyFilePath, "test_copy_from.json");
         execute("copy quotes from ?", new Object[]{uriPath});
         assertEquals(3L, response.rowCount());
-        assertThat(response.duration(), greaterThanOrEqualTo(0));
+        assertThat(response.duration(), greaterThanOrEqualTo(0f));
```

Comment:
maybe we should remove these assertions from all tests beside SQLResponseTest or another IT ?